docs/operation/cluster_and_service_configuration.rst

```diff
@@ -70,18 +75,15 @@ To add a new static user, perform the following operations:
 
 #. Save and apply the changes.
```

Comment:
isn't it save to apply the change? I mean by saving, the change will be applied right? otherwise, you would need to explain how to apply the change

Reply:
Ok, will keep it simple => `Save the changes`

Reply:
Done